packages/react-aria-components/src/Table.tsx

```diff
@@ -312,7 +313,18 @@ export interface TableProps extends Omit<SharedTableProps<any>, 'children'>, Sty
   /** Handler that is called when a user performs an action on the row. */
   onRowAction?: (key: Key) => void,
   /** The drag and drop hooks returned by `useDragAndDrop` used to enable drag and drop behavior for the Table. */
-  dragAndDropHooks?: DragAndDropHooks
+  dragAndDropHooks?: DragAndDropHooks,
+  // TODO: finalize these
+  // Perhaps we should make isLoading be loadingKeys for the eventual section/nested rows loading case
+  // Or should it follow the loadingState api we have for our async hook
+  // TODO: should the below be loadMoreOptions? These are just passed to the hook and making a prop that takes in a object containing the
```

Comment:
seems reasonable, are any of them dependent on each other? as in, if one is provided, another must be?

Reply:
Devon and I discussed this some more, for now we will be pulling these out entirely and having the user call the useLoadMore hook outside of Table. This emulates how things like useDragAndDrop are called and we can always add these back in the future if need be